frameworks/elastic/universe/config.json

```diff
@@ -340,6 +340,11 @@
           "type": "string",
           "default": ""
         }, 
+        "repositories_url_allowed_urls": {
+          "description": "URL Repository supports the following protocols: \"http\", \"https\", \"ftp\", \"file\" and \"jar\". URL repositories with http:, https:, and ftp: URLs has to be whitelisted by specifying allowed URLs in the repositories_url_allowed_urls setting. This setting supports wildcards in the place of host, path, query, and fragment.",
```

Comment:
Should be "URLs have to be whitelisted". I'm aware it's "URLs has to be whitelisted" in the official Elastic docs. ;( I'm OK with fixing this and other documentation nits in a future commit.